client/src/components/web-extension/Common/LoginModal/LoginModalForm.jsx

```diff
@@ -15,7 +15,7 @@ const LoginModalConfirm = ({isOpen, onLoginSuccess, title = ''}) => {
 					<div className="cd_we_create-wallet-layout--body">
 						<AuthLogin
 							onLoginSuccess={onLoginSuccess}
-							passwordLabel="Please login again to continue using Casper Wallet."
+							passwordLabel="Please login again to continue using the Wallet."
```

Comment:
Minor: I notice some removal of "Casper" word in text/label. Can you explain more on this action? Does it affect Casper Wallet brand?

Reply:
We're not Casper wallet @duc-q , our name is name is CasperDash and there is another Casper wallet in the market.